app/src/main/kotlin/org/fossify/calendar/helpers/Config.kt

```diff
@@ -180,16 +183,34 @@ class Config(context: Context) : BaseConfig(context) {
         get() = prefs.getBoolean(USE_PREVIOUS_EVENT_REMINDERS, true)
         set(usePreviousEventReminders) = prefs.edit().putBoolean(USE_PREVIOUS_EVENT_REMINDERS, usePreviousEventReminders).apply()
 
+    fun getDefault(reminder: String): Int {
```

Comment:
Needs a default parameter so that reminder1 can default to 10 minutes when permission is enabled.